config/deploy.rb

```diff
@@ -86,10 +86,17 @@
   desc 'Modify permissions on /srv/idp'
   task :mod_perms do
     on roles(:web), in: :parallel do
-      execute :sudo, :chown, '-R', 'ubuntu:nogroup', deploy_to
+      execute :sudo, :chown, '-R', 'ubuntu:nogroup', deploy_to + '/current', '/srv/idp/shared'
```

Comment:
this change makes *fewer* files because it is acting on `/srv/idp/current /srv/idp/shared` rather than all of `/srv/idp`.